packages/website/versioned_docs/version-legacy/faq.md

```diff
@@ -0,0 +1,95 @@
+
+We host the DocSearch data on Algolia's servers, with replications around the globe. You can find more details about the actual [server specs here][4], and more complete information in our [privacy policy][5].
+
+## Can I use DocSearch on non-doc pages?
+
+The free DocSearch we provide will crawl open-source documentation project pages or technical blog. To use it on other parts of your website, you'll need to create your own Algolia account and either:
+
+- Run the [DocSearch crawler][6] on your own
+- Use one of our other [framework integrations or API clients][7]
+
+## Can you index code samples?
+
+Yes, but we do not recommend it.
+
+Code samples are a great way for humans to understand how people use a specific method. It often requires boilerplate code though, repeated across examples, which adds noise to the results.
+
+What we recommend instead is to exclude the code blocks from the indexing (by using the `selectors_exclude` option in your config), and instead structure your content so the method names are present in the headers.
+
+## Why do I have duplicate content in my results?
+
+This can happen when you have more than one URL pointing to the same content, for example with `./docs`, `./docs/` and `./docs/index.html`.
+
+You set the `stop_urls` to all the patterns you want to exclude. The following example will exclude all URLs ending with `/` or `index.html`
+
+```json
+{
+  "stop_urls": ["/$", "/index.html$"]
+}
+```
+
+## Why are custom changes from the Algolia dashboard ineffective?
+
+Changing your setting from the dashboard might be something you want to do for some reasons .
+
+Every successful crawl sets the DocSearch settings. These settings will be overridden at the next crawl. We **do not recommend to edit anything from the dashboard**. These changes come from the JSON configuration itself.
+
+You can use the [custom_settings parameter][8] for such purpose.
+
+## A documentation website I like does not use DocSearch. What can I do?
+
+We'd love to help!
+
+If one of your favorite tool documentation websites is missing DocSearch, we encourage you to file an issue in their repository explaining how DocSearch could help. Feel free to [send us an email][1] as well, and we'll provide all the help we can.
+
+## How many records does the DocSearch crawl create?
+
+The [`nb_hits` property][8] in your configuration keeps track of the number of records the crawl has extracted and indexed by the last DocSearch run. A crawl updates this number automatically.
+
+The DocSearch scraper follows [the recommended atomic-reindexing strategy][9]. It creates a brand new temporary index to populate the data scraped from your website. When successful, the crawl overwrites the old index defined in your configuration with the key `index_name`.
+
+## Why aren't my pages indexed?
+
+We are scraping your website according to your configuration. It might happen that some pages are missing from the search. Some possible reasons for that are:
```

Comment:
```suggestion
We are scraping your website according to your configuration. It might happen that some pages are missing from the search. Some possible reasons are:
```